### ISSUE TITLE
Adapter get password issue

### ISSUE THREAD

**Issue by DeadBoolean** [iobroker-community-adapters/ioBroker.roomba#11]
The Adapter seems to receive the password wrong on initialization routine. 
Add additional character is added at the end of the recieved password. 

Is is related to the i7/i7+ and Braava devices . Maybe other devices are affected, too. 

Workarround: 
At time, to get it work, just show the password in plain text and remove the last character. 

Regards

**Comment by fjosepose** [comment]
Hi. 
Did you find a way out of this? During setup of my i7+ everything is fine and all four steps in setup are ok. When I save the instance and exit, the status goes red and the log reports that user or password is wrong.

I tried to remove last character in password...no luck

**Comment by DeadBoolean** [comment]
Hello,

 

yes, the automatic password receive function is buggy (even the on for the console) 

It’s adding an additional char at the end. 

Remove this one and it should work.

 

Von: Torbjørn Nesheim <notifications@github.com> 
Gesendet: Donnerstag, 17. September 2020 11:38
An: iobroker-community-adapters/ioBroker.roomba <ioBroker.roomba@noreply.github.com>
Cc: DeadBoolean <sascha@wedler-online.de>; Author <author@noreply.github.com>
Betreff: Re: [iobroker-community-adapters/ioBroker.roomba] Adapter get password issue (#11)

 

Hi.
Did you find a way out of this? During setup of my i7+ everything is fine and all four steps in setup are ok. When I save the instance and exit, the status goes red and the log reports that user or password is wrong.

I tried to remove last character in password...no luck

**Comment by fjosepose** [comment]
Thanks for reply. I tried to remove last character in password, but no luck:-(
Btw, my 980 works fine with the adapter.

**Comment by fjosepose** [comment]
Hiagain. For some reason the instance apparently came online. This extra character...was it a blank space? I had an x with I removed...no luck...then I put i back...and then I came online

**Comment by DeadBoolean** [comment]
No, I think it is an nonvisible/special char. 

Follow this guide to retrieve the pwd:

https://github.com/koalazak/dorita980#how-to-get-your-usernameblid-and-password

Remove the last char. 

 

This works for me. 

 

 

Von: Torbjørn Nesheim <notifications@github.com> 
Gesendet: Donnerstag, 17. September 2020 14:27
An: iobroker-community-adapters/ioBroker.roomba <ioBroker.roomba@noreply.github.com>
Cc: DeadBoolean <sascha@wedler-online.de>; Author <author@noreply.github.com>
Betreff: Re: [iobroker-community-adapters/ioBroker.roomba] Adapter get password issue (#11)

 

Hiagain. For some reason the instance apparently came online. This extra character...was it a blank space? I had an x with I removed...no luck...then I put i back...and then I came online
